modules/apps/list-type/list-type-test/src/testIntegration/java/com/liferay/list/type/service/test/ListTypeEntryServiceTest.java

```diff
@@ -142,6 +157,82 @@ public void testGetListTypeEntryByExternalReferenceCode() throws Exception {
 		_testGetListTypeEntryByExternalReferenceCode(_user);
 	}
 
+	@Test
+	public void testGetOrAddEmptyListTypeEntry() throws Exception {
+
+		// Lazy referencing disabled
+
+		Role role = RoleTestUtil.addRole(RoleConstants.TYPE_REGULAR);
+
+		RoleTestUtil.addResourcePermission(
+			role, _listTypeDefinition.getModelClassName(),
+			ResourceConstants.SCOPE_COMPANY,
+			String.valueOf(TestPropsValues.getCompanyId()), ActionKeys.UPDATE);
+
+		User user = UserTestUtil.addUser();
+
+		UserLocalServiceUtil.addRoleUser(role.getRoleId(), user.getUserId());
+
+		_setUser(user);
```

Comment:
Can we use `TestPropsValues.getUser()` to simplify this?
I just noticed that in the setup method we’re already calling _setUser(_user), so we wouldn’t need to create a user at the beginning of this test: just use _user.

Reply:
can you check if we need to use long userId1 = user.getUserId(); or we could inline the _user.getUserId()?

Reply:
Hi,
I changed my mind. Lets use the admin user, okay?
I saw that other teams used and we also sent PR related to this using the admin user :+1: